central/plugins/org.jboss.tools.central/schemas/org.jboss.tools.central.proxyWizard.exsd

```diff
@@ -111,6 +111,13 @@
                </documentation>
             </annotation>
          </attribute>
+         <attribute name="category" type="string">
+            <annotation>
+               <documentation>
+                  
```

Comment:
I assume this field doc would say something like "Installation Category of the wizard. If the category is 'earlyaccess' it will only be loaded when users have enabled earlyaccess in central".

Also, doesn't category mean something else in normal wizard sense ? no chance of a confusing overlap here?